src/components/projects/wbs-details/work-package-container/work-package-container.module.css

```diff
@@ -1,5 +1,4 @@
 /*
  * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
  * See the LICENSE file in the repository root folder for details.
- */
-
+ */
```

Comment:
This is also apparently a merge conflict, I would recommend letting prettier run on the file, so hopefully it would end up not changed